applications/StructuralMechanicsApplication/tests/test_perfect_plasticity_implementation_verification.py

```diff
@@ -246,31 +246,42 @@ def _create_reference_solution(current_model):
 
     return out
 
-def _create_post_process(main_model_part, constitutive_law_type):
-    from gid_output_process import GiDOutputProcess
-    gid_output = GiDOutputProcess(main_model_part,
-                                "gid_output_" + constitutive_law_type,
-                                KratosMultiphysics.Parameters("""
-                                    {
-                                        "result_file_configuration" : {
-                                            "gidpost_flags": {
-                                                "GiDPostMode": "GiD_PostBinary",
-                                                "WriteDeformedMeshFlag": "WriteUndeformed",
-                                                "WriteConditionsFlag": "WriteConditions",
-                                                "MultiFileFlag": "SingleFile"
-                                            },
-                                            "nodal_results"       : ["DISPLACEMENT"],
-                                            "gauss_point_results" : ["VON_MISES_STRESS","GREEN_LAGRANGE_STRAIN_VECTOR","PK2_STRESS_VECTOR","PLASTIC_DISSIPATION","PLASTIC_STRAIN","EQUIVALENT_PLASTIC_STRAIN","PLASTIC_STRAIN_VECTOR","UNIAXIAL_STRESS"]
+def _create_post_process(main_model_part, constitutive_law_type, debug = "GiD"):
+    if debug == "GiD":
+        from gid_output_process import GiDOutputProcess
+        output = GiDOutputProcess(main_model_part,
+                                    "output_" + constitutive_law_type,
+                                    KratosMultiphysics.Parameters("""
+                                        {
+                                            "result_file_configuration" : {
+                                                "gidpost_flags": {
+                                                    "GiDPostMode": "GiD_PostBinary",
+                                                    "WriteDeformedMeshFlag": "WriteUndeformed",
+                                                    "WriteConditionsFlag": "WriteConditions",
+                                                    "MultiFileFlag": "SingleFile"
+                                                },
+                                                "nodal_results"       : ["DISPLACEMENT"],
+                                                "gauss_point_results" : ["VON_MISES_STRESS","GREEN_LAGRANGE_STRAIN_VECTOR","PK2_STRESS_VECTOR","PLASTIC_DISSIPATION","PLASTIC_STRAIN","EQUIVALENT_PLASTIC_STRAIN","PLASTIC_STRAIN_VECTOR","UNIAXIAL_STRESS"]
+                                            }
                                         }
-                                    }
-                                    """)
-                                )
+                                        """)
+                                    )
+    elif debug == "VTK":
+        from vtk_output_process import VtkOutputProcess
```

Comment:
Please do full imports, this will stop working 1. October

Reply:
Well, in that case, we need to update maaaaany test

Reply:
I know :)